Signal/src/ViewControllers/DebugUI/DebugUIBackup.m

```diff
@@ -44,6 +44,10 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
                                      actionBlock:^{
                                          [DebugUIBackup logDatabaseSizeStats];
                                      }]];
+    [items addObject:[OWSTableItem itemWithTitle:@"Clear All CloudKit Records"
+                                     actionBlock:^{
+                                         [DebugUIBackup clearAllCloudKitRecords];
+                                     }]];
```

Comment:
App doesn't retain any local state about whether or not there is a valid backup.  Every new backup export/import consults CloudKit, the "single point of truth" for current backup state.  So clearing CloudKit clears everything.